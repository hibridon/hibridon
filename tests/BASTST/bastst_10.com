inp=bastst_10.inp
out=bastst_10.out
run
save=save_10.inp