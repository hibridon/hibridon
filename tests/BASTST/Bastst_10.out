      BASTST                                     
 DUMMY POTENTIAL FOR TESTING PURPOSES            
 ==============================
     *** Parameters:
 JTOT1  =  20        JTOT2  =  20        JTOTD  =   5        JLPAR  =   1
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  48        IPRINT =   1
 FSTFAC =  15.00     RINCR  =  1.000     RCUT   =  30.00     RENDAI =  1.000    
 RENDLD =  1.000     RSTART =  1.000     SPAC   = 0.1000     TOLAI  =  1.000    
 XMU    =  16.47    
     ***  system parameters:
 NTERM  =   1        NPHOTO =   1        NVIB   =   0        IBRAN  =   0
 A-SO   =  881.0    
 LAMMIN:    1
 LAMMAX:   13
 MPROJ:     0
     *** Flags:
 AIRYFL= F    BASTST= T    BATCH = F    CHLIST= T    CSFLAG= F    FLAGHF= T
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= F    NOPRIN= F    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= F    WAVEFL= F
 WRPART= F    WRSMAT= F    WRXSEC= F    BOUNDC= F
 ** Energies:     5000.000000
 NOUT: -2; JOUT:  0   2
 INDOUT:    0   1
 ==============================

 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V 5.00

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
CMAKE BUILD DATE: 2023-01-24 at 10:18
CMAKE BUILD TYPE: Debug
CODE GIT REVISION: 6f4ce0f461f00ddde838e7db90340246d61a8a35
SYS: Darwin 22.2.0 x86_64
COMPILER: /opt/intel/oneapi/compiler/2022.2.1/mac/bin/intel64/ifort
COMPILE OPTIONS: 
 --------------------------------------------------------------------------
            CURRENT HARDWARE CONFIGURATION:
 
      Model Name: MacBook Pro
      Model Identifier: MacBookPro14,3
      Processor Name: Quad-Core Intel Core i7
      Processor Speed: 2.9 GHz
      Number of Processors: 1
      Total Number of Cores: 4
      L2 Cache (per Core): 256 KB
      L3 Cache: 8 MB
      Hyper-Threading Technology: Enabled
      Memory: 16 GB
 --------------------------------------------------------------------------

 -----------------------------------------------------------------------------
 All publications resulting from use of the integrators included in the
 Hibridon code must include the following reference:

 D. E. Manolopoulos, J. Chem. Phys. 85, 6425 (1986); M. H. Alexander and D. E.
 Manolopoulos, J. Chem. Phys. 80, 2044 (1987).

 All publications involving the determination ofphotodissociation cross sections
 must also include the following reference:

 M. H. Alexander, Comput. Phys. Commun, 75, 87 (1993).

 All publications investigating flux redistribution must include the following references:

 M. H. Alexander, J. Chem. Phys. 95, 8931 (1991); D. E. Manolopoulos and
 M. H. Alexander, J. Chem. Phys.  97, 2527 (1992).

 All publications resulting from use of the Hibridon package must include
 the following reference:

 HIBRIDON is a package of programs for the time-independent quantum treatment
 of inelastic collisions and photodissociation written by M. H. Alexander,
 D. E.  Manolopoulos, H.-J. Werner, and B. Follmeg, with contributions by
 P. F. Vohralik, D. Lemoine, G. Corey, B. Johnson, T. Orlikowski, A. Berning,
 A. Degli-Esposti, C. Rist, P. Dagdigian, B. Pouilly, G. van der Sanden, M. Yang, F. de Weerd, S. Gregurick, J. Klos, and F. Lique
 -----------------------------------------------------------------------------

 ** LABEL:          BASTST                                     
 ** POT NAME:  DUMMY POTENTIAL FOR TESTING PURPOSES            
 ** DATE:      Jan 24 10:20:00 2023

 *** WARNING:  NAIRY .NE. 1 BUT AIRYFL .EQ. .FALSE.
 ** NMAX= 500  NAIRY= 500
 ** TOTAL NUMBER OF ANISOTROPIC TERMS=14

 **  CC 2/2 ATOM ** RMU=  16.4700       E=  5000.00   JTOT=   20   JLPAR= 1

 OPEN CHANNELS ELIMINATED WHICH ARE CLOSED AT R=   30.00

   N    J   J12    L      EINT(CM-1)
   1  0.5     0   20       881.000
   2  0.5     1   20       881.000
   3  1.5     1   20      -440.500
   4  1.5     2   18      -440.500
   5  1.5     2   20      -440.500
   6  1.5     2   22      -440.500

 ILAM  LAMBDA  ICOL  IROW   I    IV2    VEE
ILAM=  1 LAMNUM(ILAM) =      0
   2      2      6     6     1    1.0000000    
ILAM=  2 LAMNUM(ILAM) =      1
   3      3      1     1     2    1.0000000    
   3      3      5     5     3    1.0000000    
ILAM=  3 LAMNUM(ILAM) =      2
   4      4      2     2     4    1.0000000    
   4      4      3     3     5    1.0000000    
   4      4      4     4     6    1.0000000    
ILAM=  4 LAMNUM(ILAM) =      3
ILAM=  5 LAMNUM(ILAM) =      0
   6      6      6     6     7    1.0000000    
ILAM=  6 LAMNUM(ILAM) =      1
   7      7      1     1     8    1.0000000    
   7      7      5     5     9    1.0000000    
ILAM=  7 LAMNUM(ILAM) =      2
   8      8      2     2    10    1.0000000    
   8      8      3     3    11    1.0000000    
   8      8      4     4    12    1.0000000    
ILAM=  8 LAMNUM(ILAM) =      3
ILAM=  9 LAMNUM(ILAM) =      0
  10     10      1     2    13    40.987803    
  10     10      1     3    14   -2.8284271    
  10     10      1     4    15    40.890097    
  10     10      2     5    16   -4.0000000    
  10     10      3     5    17    57.965507    
ILAM= 10 LAMNUM(ILAM) =      5
  11     11      2     2    18   0.50000000    
  11     11      3     6    19  -0.50000000    
  11     11      4     4    20  -0.50000000    
ILAM= 11 LAMNUM(ILAM) =      3
ILAM= 12 LAMNUM(ILAM) =      0
  13     13      1     3    21   0.70710678    
  13     13      1     6    22  -0.70710678    
  13     13      2     5    23    1.0000000    
ILAM= 13 LAMNUM(ILAM) =      3
 ** TOTAL NUMBER OF NONZERO V2 MATRIX ELEMENTS IS    23
 ** BASTST=.TRUE.; TEST OF SUBROUTINE BASIS
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:     0
      TIMING:  ELAPSED 00:00:01 / CPU 00:00:00 / MP RATIO    0.2 %
      CURRENT DATE:  Jan 24 10:20:00 2023
 ===============================================================================
